web/src/components/__tests__/CityNotCooperatingFooter.spec.tsx

```diff
@@ -11,7 +11,6 @@ jest.mock('../../constants/buildConfig', () =>
       cityNotCooperating: true,
     },
     legacyLightTheme,
-    lightTheme,
```

Comment:
🤔 Why did you remove this/had to remove this?

Reply:
because I'm not using `lightTheme` at `ThemeContainer.tsx` anymore.